release-version.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/sh
+
+# ============================================================================
+# release-version.sh
```

Comment:
This script shouldn't be part of this repository. Instead, it's recommended to always use the latest version in `scijava-scripts`.